pom.xml

```diff
@@ -4,7 +4,7 @@
 
     <groupId>org.disq-bio</groupId>
     <artifactId>disq</artifactId>
-    <version>0.3.6-SNAPSHOT</version>
+    <version>0.3.7-SNAPSHOT</version>
```

Comment:
I think you accidentally included the snapshot version change here.  Could you revert that?

Reply:
Sure, but I thought we'd want to bump the version with this PR.

Reply:
It gets bumped automatically as part of the release process.  If we bump it manually we'll end up skipping a version.